src/main/java/hudson/plugins/synergy/SynergySCM.java

```diff
@@ -120,37 +120,14 @@ private DescriptorImpl() {
 
 		@Override
 		public boolean configure(StaplerRequest request, JSONObject formData) throws FormException {
-			ccmExe = request.getParameter("synergy.ccmExe");
-			ccmUiLog = request.getParameter("synergy.ccmUiLog");
-			ccmEngLog = request.getParameter("synergy.ccmEngLog");
-			pathName = request.getParameter("synergy.pathName");
+			ccmExe = formData.getString("synergy.ccmExe");
+			ccmUiLog = formData.getString("synergy.ccmUiLog");
+			ccmEngLog = formData.getString("synergy.ccmEngLog");
+			pathName = formData.getString("synergy.pathName");

```

Comment:
If appropriate setters are defined, `StaplerRequest.bindJSON(Object, JSONObject)` may work here and be a bit more standard. Indeed that is what the `super` call would do, so if your form binding is fully normal, you should just be able to delete this method.